content/300-guides/200-deployment/110-deployment-guides/500-deploying-to-netlify.mdx

```diff
@@ -116,52 +140,64 @@ postgresql://__USER__:__PASSWORD__@__HOST__/__DATABASE__
 For example:
 
 ```
-postgresql://janedoe:randompassword@yourpostgres.compute-1.amazonaws.com:5432/yourdbname
+postgresql://janedoe:randompassword@db.projectcode.supabase.co:5432/postgres
 ```
 
 ## 4. Set the <inlinecode>DATABASE_URL</inlinecode> variable in a <inlinecode>.env</inlinecode> file
 
 You can store local environment variables related to Prisma in an `.env` file. For example, instead of setting the database connection string as the local environment variable `DATABASE_URL`, you can keep it in an `.env` file.
 
-With the help of the `dotenv` library, Prisma CLI loads the variables defined in the `.env` file into the environment. Later, when you run `prisma db push`, the command reads the environment variables.
+### Steps
 
-Create the `.env` file at the root of your project folder. The example below shows an example of the directory listing of the root project folder that contains the `.env` file.
+1. Create the `.env` file at the root of your project folder.
 
-<CodeWithResult expanded={true}>
+   ```terminal
+   touch .env
+   ```
 
-<cmd>
+   The example below shows the directory listing of the root project folder that contains the `.env` file.
 
-```terminal
-ls -1a
-```
+   <CodeWithResult expanded={true}>
 
-</cmd>
-<cmdResult>
+   <cmd>
 
-```code no-copy
-.env
-.git/
-README.md
-functions/
-node_modules/
-package-lock.json
-package.json
-prisma/
-```
+   ```terminal
+   ls -1a
+   ```
 
-</cmdResult>
-</CodeWithResult>
+   </cmd>
+
+   <cmdResult>
+
+   ```code no-copy
+   .env
+   .git/
+   README.md
+   functions/
+   node_modules/
+   package-lock.json
+   package.json
+   prisma/
+   ```
+
+   </cmdResult>
+
+   </CodeWithResult>
 
 1. Create the `.env` file in the local repository folder.
-2. Add a line that defines `DATABASE_URL` with your database connection string.
+1. Add a line that defines `DATABASE_URL` with your database connection string.
 
    ```bash file=.env
    DATABASE_URL="postgresql://janedoe:randompassword@yourpostgres.compute-1.amazonaws.com:5432/yourdbname"
```

Comment:
In current state this can be updated to look more like a Supabase connection string.